File: java/arrays/check_sorted.java
import java.util.Scanner;

public class CheckSortedBoth {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter size of array: ");
        int n = sc.nextInt();

        if (n == 0) {
            System.out.println("Array is empty!");
            return;
        }

        int[] arr = new int[n];
        System.out.println("Enter " + n + " elements:");
        for (int i = 0; i < n; i++) {
            arr[i] = sc.nextInt();
        }

        boolean ascending = true;
        boolean descending = true;

        for (int i = 0; i < n - 1; i++) {
            if (arr[i] > arr[i + 1]) ascending = false;
            if (arr[i] < arr[i + 1]) descending = false;
        }

        if (ascending) {
            System.out.println("Array is sorted in ascending order.");
        } else if (descending) {
            System.out.println("Array is sorted in descending order.");
        } else {
            System.out.println("Array is NOT sorted.");
        }
    }
}

File: python/arrays/check_sorted.py
n = int(input("Enter size of array: "))
arr = list(map(int, input(f"Enter {n} elements: ").split()))

ascending = descending = True
for i in range(n-1):
    if arr[i] > arr[i+1]:
        ascending = False
    if arr[i] < arr[i+1]:
        descending = False

if ascending:
    print("Array is sorted in ascending order.")
elif descending:
    print("Array is sorted in descending order.")
else:
    print("Array is NOT sorted.")
